st.info

```diff
@@ -1,4 +1,5 @@
 xst-mono| simpleterm monocolor,
+    Su,
```

Comment:
nitpick -- whitespace should match other lines/tabs (same thing with the `int ustyle;` declaration, and `int carg`)